updatemodules/partitions/utils/efi/efi.go

```diff
@@ -63,18 +63,13 @@ import (
 
 const preallocatedItemSize = 10
 
-const (
-	hdFormatPCAT = iota + 1
-	hdFormatGPT
-)
-
 const efiBootAbbrevHD = 2
 const eddDefaultDevice = 0x80
 
 const (
-	hdSignatureNone = iota
-	hdSignatureMBR
-	hdSignatureGUID
+	//hdSignatureNone = iota
```

Comment:
I don't like this fix. If there is no way to suppress this warning:
```go
        // hdSignatureNone = 0
	// hdSignatureMBR = 1
	hdSignatureGUID = 2
```

Reply:
added warning suppresen  (//nolint)